src/Entity/Programme.php

```diff
@@ -34,17 +34,31 @@ public function getStartDate(): \DateTime
         return $this->startDate;
     }
 
-    public function setStartDate($startDate): self
+    public function setStartDate(\DateTime $startDate): self
     {
-        return $this->startDate = $startDate;
+        $this->startDate = $startDate;
```

Comment:
Not something wrong, but in the Building Class it was startTime, now it's startDate.